dissect/hypervisor/descriptor/vbox.py

```diff
@@ -11,7 +11,8 @@ def __init__(self, fh: IO):
         self._xml: Element = ElementTree.fromstring(fh.read())
 
     def disks(self) -> Iterator[str]:
-        for hdd_elem in self._xml.findall(
-            f".//{self.VBOX_XML_NAMESPACE}HardDisk[@location][@format='VDI'][@type='Normal']"
-        ):
-            yield hdd_elem.attrib["location"]
+        for hdd_elem in self._xml.findall(f".//{self.VBOX_XML_NAMESPACE}HardDisk[@location][@type='Normal']"):
+            # Allow format specifier to be case-insensitive (i.e. VDI, vdi)
+            format_attr = hdd_elem.get("format")
```

Comment:
For reviewer: there is no support for the 'translate` xpath function in `xml.etree`, so I do the check "in code"